### PR TITLE
controllers now using the database

### PR DESCRIPTION
**Pull request by bozsAdam**

### CODE REVIEW

File: backend/src/main/java/hu/codecool/flatium/flatmanager/controller/BuildingController.java [hunk 1]
```diff
@@ -17,36 +15,49 @@
...
     @GetMapping(path = "/get-buildings")
-    public ResponseEntity<List<Building>> getAllBuildings() {
-        return ResponseEntity.ok(buildingStorage.getBuildings());
+    public List<Building> getAllBuildings() {
+        return buildingRepository.findAll();
+    }
+
+    @PostMapping(path = "get-building")
+    public Building getBuilding(@RequestBody int id){
+        return buildingRepository.findById(id).orElseThrow(()->new IllegalStateException("anyád"));
```
Comment (doraKoreny) [inline backend/src/main/java/hu/codecool/flatium/flatmanager/controller/BuildingController.java:42]: Please write another exception message.

File: backend/pom.xml [hunk 2]
```diff
@@ -55,7 +55,6 @@
             <artifactId>postgresql</artifactId>
             <version>42.2.5</version>
         </dependency>
-
```
Comment (Janimarton) [inline backend/pom.xml:58]: It was really necessary that minus line? You should Ctrl + Alt + L (autoformat) your files always before commiting.

File: backend/src/main/java/hu/codecool/flatium/flatmanager/controller/FlatController.java [hunk 3]
```diff
@@ -4,51 +4,71 @@
...
+    private FlatRepository flatRepository;
 
     @Autowired
-    private FlatUserStorageService flatUserStorage;
+    private FlatUserRepository flatUserRepository;
 
     @PostMapping(path = "/create-flat")
-    public ResponseEntity<Flat> createFlat() {
-        return ResponseEntity.ok(flatStorage.addFlat());
+    public Flat createFlat() {
```
Comment (doraKoreny) [inline backend/src/main/java/hu/codecool/flatium/flatmanager/controller/FlatController.java:28]: This method needs to be refactored, use the request body.

File: backend/src/main/java/hu/codecool/flatium/flatmanager/controller/FlatController.java [hunk 4]
```diff
@@ -4,51 +4,71 @@
...
 
     @PostMapping(path = "/add-to-flat")
-    public ResponseEntity<String> getFlatUser(@RequestBody BoughtFlatRequest boughtFlatRequest) {
-        Person person = flatUserStorage.getUserById(boughtFlatRequest.getUserId());
-        Flat flat = flatStorage.getFlat(boughtFlatRequest.getFlatId());
-        flat.setFlatUser(person);
-        flatStorage.updateFlat(flat);
-        return ResponseEntity.ok("User " + person.getName() + " successfully alligned to flat.");
+    public String addToFlat(@RequestBody BoughtFlatRequest boughtFlatRequest) {
+
```
Comment (doraKoreny) [inline backend/src/main/java/hu/codecool/flatium/flatmanager/controller/FlatController.java:61]: Nothing serious. There are a lot of empty lines in this method.

**Pull request closed without merging**